### DISCUSSION TITLE
Add an error for S3 bucket name max length.

### DISCUSSION THREAD

**Discussion by davimmt** [terraform-linters/tflint-ruleset-aws#539] [Ideas]
Add an error for S3 bucket  name max length.

![image](https://github.com/terraform-linters/tflint/assets/53799940/a3dbd2a0-a43f-4a97-b9d2-a107aa3b45a8)

I don't know if this is whithin tflint's scope, however I wanted your opinion on this.

**Comment by wata727** [comment]
I think this is a useful rule. PR is welcome.

**Reply by davimmt** [reply]
Forgot to mention it here. 

https://github.com/terraform-linters/tflint-ruleset-aws/pull/554
